ring_theory/algebra.lean

```diff
@@ -0,0 +1,495 @@
+/-
+Copyright (c) 2018 Kenny Lau. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kenny Lau
+
+Algebra over Commutative Ring (under category)
+-/
+
+import data.polynomial
+import linear_algebra.multivariate_polynomial
+import linear_algebra.tensor_product
+import ring_theory.subring
+
+universes u v w u₁ v₁
+
+open lattice
+
+infix ` ⊗ `:100 := tensor_product
```

Comment:
global infix? Does this cause problems with the use of `tensor_product` in `linear_map`? Or is this the same one